File: FizzBuzz/fizz_buzz.rb
(1..50).each do |number|
  if number % 3 == 0 and number % 5 == 0  
    puts "FizzBuzz"
  elsif number % 3 == 0 
    puts "Fizz"
  elsif number % 5 == 0 
    puts "Buzz"
  else
    puts number
  end
end  

File: FizzBuzz/FizzBuzz.kt
fun fizzBuzz(number: Int): String {
    return when {
        number % 3 == 0 && number % 5 == 0 -> "FizzBuzz"
        number % 3 == 0 -> "Fizz"
        number % 5 == 0 -> "Buzz"
        else -> number.toString()
    }
}

fun main() {
    for (i in 1..100) {
        println(fizzBuzz(i))
    }
}